integration_test/third_party_apps_test/applications/mysql/metadata.yaml

```diff
@@ -32,11 +32,28 @@ minimum_supported_agent_version:
   logging: 2.5.0
 supported_operating_systems: linux
 platforms_to_skip:
-  # MySQL is not currently supported on various distros.
+  # MySQL is not currently supported on various distros.  - ml-images:common-gpu-debian-11-py310
+  - debian-cloud:debian-11
   - debian-cloud:debian-12
   - debian-cloud:debian-12-arm64
+  - ml-images:common-gpu-debian-11-py310
+  - rocky-linux-cloud:rocky-linux-8
+  - rocky-linux-cloud:rocky-linux-8-optimized-gcp-arm64
+  - rocky-linux-cloud:rocky-linux-9
+  - rocky-linux-cloud:rocky-linux-9-arm64
+  - ubuntu-os-cloud:ubuntu-2004-lts
+  - ubuntu-os-cloud:ubuntu-2004-lts-arm64
+  - ubuntu-os-cloud:ubuntu-2204-lts
+  - ubuntu-os-cloud:ubuntu-2204-lts-arm64
+  - ubuntu-os-cloud:ubuntu-2404-lts-amd64
+  - ubuntu-os-cloud:ubuntu-2404-lts-arm64
+  - ubuntu-os-cloud:ubuntu-2410-amd64
+  - ubuntu-os-cloud:ubuntu-minimal-2410-amd64
+  - ubuntu-os-cloud:ubuntu-2410-arm64
+  - ubuntu-os-cloud:ubuntu-minimal-2410-arm64
+  - suse-cloud:sles-12
   - suse-cloud:sles-15-arm64
-supported_app_version: ["5.7", "8.0"]
+supported_app_version: ["5.7", "8.0", "8.4"]
```

Comment:
This bit will update our public docs; I would hold off making this change until we've finished adding the other (Debian, Ubuntu, RH, etc.) distros to the tests. (Those other distros can be added in follow-up PRs later since this one was just focused on SLES 15.)